certbot/tests/error_handler_test.py

```diff
@@ -47,6 +47,10 @@ def setUp(self):
         self.handler = error_handler.ErrorHandler(self.init_func,
                                                   *self.init_args,
                                                   **self.init_kwargs)
+        self.exit_handler = error_handler.ExitHandler(self.init_func,
```

Comment:
This is definitely a nit, but it might be kind of nice to:

1. Have a separate test class for `ExitHandler` to simplify the classes and make it clear what functionality is tested where.
2. Ensure all of the other functionality of `ExitHandler` works now and in the future.

Doing this might be a bit tricky due to the way tests are automatically found and run, but one way would be to:

1. Make a `ExitHandlerTest` class that inherits from this class.
2. In `ExitHandlerTest`, set `self.handler` to an `ExitHandler`.
3. Change the name of the methods you're adding to something like `test_regular_exit` and override this in `ExitHandler`.

This is only a suggestion so feel free to push back and/or ignore it, but I think it should be pretty easy and might be worth it.